spec/system/admin/products_v3/update_spec.rb

```diff
@@ -439,6 +441,14 @@
         expect(page).not_to have_css('form.disabled-section#filters')
       end
 
+      describe "producer" do
```

Comment:
shouldn't we mark this one as pending ?

Reply:
Yeah I wasn't sure how to best to document these.
You can't add pending to a `describe` or `context` block, only `it`.
If you add an `it` without any code, it passes by default. So pending wouldn't work.

Maybe it would have been better to use `xdescribe` to signify everything inside it is skipped for now. Then the `it`s can be uncommented. Happy to change that if you think it's better.

Reply:
I think we should use `xdescribe` here, it will come up as pending in the test result.

Reply:
If you want to code to be executed then you can also add `before { pending }`.

Reply:
Good points, I'll try one of those.